opal/mca/pmix/pmix1xx/pmix/include/private/autogen/config.h.in

```diff
@@ -0,0 +1,636 @@
+/* include/private/autogen/config.h.in.  Generated from configure.ac by autoheader.  */
```

Comment:
this file was removed from .gitignore when the pmix1xx directory was removed,
so I guess git checkout or git rebase did not remove this file,
and following git add added it to the branch